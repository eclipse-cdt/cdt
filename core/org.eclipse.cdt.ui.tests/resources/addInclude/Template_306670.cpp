void test() {
    func306670(1);
}
