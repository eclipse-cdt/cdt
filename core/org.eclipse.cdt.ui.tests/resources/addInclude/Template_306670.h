template <typename T> void func306670(T) {}
void bla() {
	func306670(1); // make sure the instance is in the index.
}
