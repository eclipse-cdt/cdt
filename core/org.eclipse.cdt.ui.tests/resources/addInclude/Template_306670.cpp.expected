#include "Template_306670.h"

void test() {
    func306670(1);
}
